impeller/entity/shaders/blending/advanced_blend.frag

```diff
@@ -41,13 +41,12 @@ void main() {
                                  v_dst_texture_coords  // texture coordinates
                                  ));
   dst *= blend_info.dst_input_alpha;
-  f16vec4 src = IPHalfUnpremultiply(
+  f16vec4 src =
       blend_info.color_factor > 0.0hf
           ? blend_info.color
```

Comment:
This change makes sense... this color never gets premultiplied on the CPU side before we get here. Could you add a comment saying this color is expected to be unpremultiplied in the shader struct where it's defined?

Reply:
That comment already exists:
https://github.com/flutter/engine/blob/main/impeller/entity/shaders/blending/advanced_blend.frag#L18